docker-storage-setup.service

```diff
@@ -5,7 +5,7 @@ Before=docker.service
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/docker-storage-setup /etc/sysconfig/docker-storage-setup /etc/sysconfig/docker-storage
+ExecStart=/usr/bin/container-storage-setup /etc/sysconfig/docker-storage-setup /etc/sysconfig/docker-storage
 EnvironmentFile=-/etc/sysconfig/docker-storage-setup
```

Comment:
Why do we need to retain this file now? Even if we retain, should we really install it during "make install" time? I feel that installing this unit file feels little odd.